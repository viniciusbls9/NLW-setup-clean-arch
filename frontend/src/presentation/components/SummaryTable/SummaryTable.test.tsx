import { describe, test, expect } from 'vitest';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { SummaryTable } from '.';

describe('SummaryTable Component', () => {
  test('Should render header component correctly', () => {
    const { container } = render(<SummaryTable />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
