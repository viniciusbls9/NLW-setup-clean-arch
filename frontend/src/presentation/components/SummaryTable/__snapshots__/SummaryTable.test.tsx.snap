// Vitest Snapshot v1

exports[`SummaryTable Component > Should render header component correctly 1`] = `
<div
  class="w-full flex"
>
  <div
    class="grid grid-rows-7 grid-flow-row gap-3"
  >
    <div
      class="text-zinc-400 text-xl h-10 w-10 font-bold flex items-center justify-center"
    >
      D
    </div>
    <div
      class="text-zinc-400 text-xl h-10 w-10 font-bold flex items-center justify-center"
    >
      S
    </div>
    <div
      class="text-zinc-400 text-xl h-10 w-10 font-bold flex items-center justify-center"
    >
      T
    </div>
    <div
      class="text-zinc-400 text-xl h-10 w-10 font-bold flex items-center justify-center"
    >
      Q
    </div>
    <div
      class="text-zinc-400 text-xl h-10 w-10 font-bold flex items-center justify-center"
    >
      Q
    </div>
    <div
      class="text-zinc-400 text-xl h-10 w-10 font-bold flex items-center justify-center"
    >
      S
    </div>
    <div
      class="text-zinc-400 text-xl h-10 w-10 font-bold flex items-center justify-center"
    >
      S
    </div>
  </div>
  <div
    class="grid grid-rows-7 grid-flow-col gap-3"
  >
    <button
      aria-controls="radix-:r0:"
      aria-expanded="false"
      aria-haspopup="dialog"
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg"
      data-state="closed"
      type="button"
    />
    <button
      aria-controls="radix-:r1:"
      aria-expanded="false"
      aria-haspopup="dialog"
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg"
      data-state="closed"
      type="button"
    />
    <button
      aria-controls="radix-:r2:"
      aria-expanded="false"
      aria-haspopup="dialog"
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg"
      data-state="closed"
      type="button"
    />
    <button
      aria-controls="radix-:r3:"
      aria-expanded="false"
      aria-haspopup="dialog"
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg"
      data-state="closed"
      type="button"
    />
    <button
      aria-controls="radix-:r4:"
      aria-expanded="false"
      aria-haspopup="dialog"
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg"
      data-state="closed"
      type="button"
    />
    <button
      aria-controls="radix-:r5:"
      aria-expanded="false"
      aria-haspopup="dialog"
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg"
      data-state="closed"
      type="button"
    />
    <button
      aria-controls="radix-:r6:"
      aria-expanded="false"
      aria-haspopup="dialog"
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg"
      data-state="closed"
      type="button"
    />
    <button
      aria-controls="radix-:r7:"
      aria-expanded="false"
      aria-haspopup="dialog"
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg"
      data-state="closed"
      type="button"
    />
    <button
      aria-controls="radix-:r8:"
      aria-expanded="false"
      aria-haspopup="dialog"
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg"
      data-state="closed"
      type="button"
    />
    <button
      aria-controls="radix-:r9:"
      aria-expanded="false"
      aria-haspopup="dialog"
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg"
      data-state="closed"
      type="button"
    />
    <button
      aria-controls="radix-:ra:"
      aria-expanded="false"
      aria-haspopup="dialog"
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg"
      data-state="closed"
      type="button"
    />
    <button
      aria-controls="radix-:rb:"
      aria-expanded="false"
      aria-haspopup="dialog"
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg"
      data-state="closed"
      type="button"
    />
    <button
      aria-controls="radix-:rc:"
      aria-expanded="false"
      aria-haspopup="dialog"
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg"
      data-state="closed"
      type="button"
    />
    <button
      aria-controls="radix-:rd:"
      aria-expanded="false"
      aria-haspopup="dialog"
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg"
      data-state="closed"
      type="button"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
    <div
      class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg opacity-40 cursor-not-allowed"
    />
  </div>
</div>
`;
